recipe/meta.yaml

```diff
@@ -22,6 +22,16 @@ build:
   # not ready yet; needs to unpin pybind11
   skip: true  # [py>=312]
   number: 1
+  # Skip all builds but the the ones for Python 3.9 on Linux
+  # TODO: is there a way to adapt the condition using `conda_build_config.yaml` so that:
```

Comment:
```suggestion
  # Skip all builds but the ones for Python 3.9 on Linux.
  # TODO: is there a way to adapt the condition but using elements defined
  # within `conda_build_config.yaml` so that:
```